### ISSUE TITLE
False positive: Intellij region comment

### ISSUE THREAD

**Issue by ipereziriarte** [arturbosch/detekt#1078]
I've done a quick check on the repo but looks like it hasn't been reported before.

It looks like if the code contains [intellij's region comments](https://www.jetbrains.com/help/idea/using-code-editor.html#using_folding_comments), detekt reports them with "Missing space after // (cannot be auto-corrected)"

## Expected Behavior
Given a region comment like

```
//region LIFECYCLE
    fun onViewResumed() {
        runDisplayCountrySpecificViewsLogic()
    }
//endregion
```
Detekt should not trigger any error

Region comments should be handled as an exception to the comment spacing rule.

## Context
This is happening to me on Android Studio 3.1.4 and Detekt version 1.0.0.RC7-3

If this issue is accepted I don't think that it could take me too much time to create a PR to add this exception to the rule.

Thoughts?

**Comment by schalkms** [comment]
Which rule reports this violation?

**Comment by schalkms** [comment]
Which rule does report this violation?

**Comment by arturbosch** [comment]
Seems related: https://github.com/shyiko/ktlint/issues/273

**Comment by ipereziriarte** [comment]
Thanks @arturbosch !
I'm using detekt-formatting, so I will follow your advice and report the issue to ktlint.

Thanks!

**Comment by schalkms** [comment]
Fixed with [ktlint/#278](https://github.com/shyiko/ktlint/issues/278)
